charts/test-guide/README.md

```diff
@@ -31,3 +31,15 @@ To see all configurable options with detailed comments, visit the chart's [value
 # Show chart values
 $ helm show values tracetronic/test-guide
 ```
+
+## Migration
+
+### To 0.4.0
+
+This version supports a configurable TEST-GUIDE application directory which must match the used image.
+
+#### values.yaml
```

Comment:
Maybe use following table structure

<table>
<tr>
	<td>Change
	<td>Name
	<td>Description
	<td>Default-Value
	<td>Value to reuse existing clusters
<tr>
	<td>Add value
	<td>`baseSettings.tgWorkspace`
	<td>TEST-GUIDE base directory
	<td>`/home/tg_user/`
	<td>`/app/`
<tr>
	<td>Change default value
	<td>`db.url`
	<td>Reference to the main database of TEST-GUIDE
	<td>`/home/tg_user/TTS-TM/TT-DB`
	<td>`/app/TTS-TM/TT-DB`
<tr>
	<td>Change default value
	<td>`dbMonitoring.url`
	<td>Reference to the monitoring database of TEST-GUIDE
	<td>`/home/tg_user/TTS-TM/TT-Monitoring`
	<td>`/app/TTS-TM/TT-DB`
<tr>
	<td>Change default value
	<td>`dbArtifact.url`
	<td>Reference to the artifact database of TEST-GUIDE
	<td>`/home/tg_user/TTS-TM/TT-Artifact`
	<td>`/app/TTS-TM/TT-Artifact`
</table>

Reply:
remarks implemented

Reply:
after rethinking this overview the description shouldn't be the documentation of the value. it should be the reason for the change:
1: make the TEST-GUIDE workspace path configurable
2-4: adjust path to the official TEST-GUIDE image